src/api.js

```diff
@@ -92,6 +92,11 @@ class API extends Exchange.API {
   authPATCH (endpoint, data) {
     return this._authRequest('PATCH', endpoint, data);
   }
+
+  DELETE (endpoint, data) {
```

Comment:
Actually I didn't use camel case for HTTP verbs in the other methods here either. We can change that later if needed.